src/lib/crud_operations.js

```diff
@@ -0,0 +1,26 @@
+import loadTask from './loader.js';
+import toDoList from './tasks.js';
+
+const CRUD = (() => {
+  const addTask = (e) => {
+    e.preventDefault();
+    const description = e.srcElement.firstChild.value;
+    const index = toDoList.toDoListArray.length;
+    loadTask(description, false, index);
+    e.srcElement.reset();
+    localStorage.setObj('myToDoList', toDoList.toDoListArray);
+  };
+
+  /*
+  const removeTask = (bookTitle) => {
+    const book = getBook(bookTitle);
+    const bookLibraryIndex = myLibrary.indexOf(book);
+    myLibrary.splice(bookLibraryIndex, 1);
+  }; */
+
```

Comment:
- [ ] Remove the commented line of code as removing commented code make codebase cleaner and nicer.